cirq-core/cirq/sim/clifford/stabilizer_sampler.py

```diff
@@ -53,7 +53,7 @@ def run_sweep(
 
     def _run(self, circuit: circuits.AbstractCircuit, repetitions: int) -> Dict[str, np.ndarray]:
 
-        measurements: Dict[str, List[int]] = {
+        measurements: Dict[str, List[np.ndarray]] = {
```

Comment:
Why does this change?

Reply:
This seems to have been incorrectly typed all along. It only worked because `log_of_measurement_results` values were `Any`. I had to change the latter so that the condition logic would work, and that highlighted this error here.